htdocs/admin/notification.php

```diff
@@ -198,6 +202,20 @@
 }
 print '</td>';
 print '</tr>';
+
+$arrayofnotifto = array(
```

Comment:
Because we need to be sure of the key, use a "named key" into array so 
```
$arrayofnotifto = array(
-1 => '',
1=> $langs->trans('notiftouser')
...
```

Reply:
ok i have delet becose i change the method